model/modules/path_classifier.py

```diff
@@ -15,8 +17,13 @@ class PathClassifier(nn.Module):
     def __init__(self, config: ClassifierConfig, out_size: int):
         super().__init__()
         self.out_size = out_size
-        self.attention = LuongAttention(config.classifier_input_size)
-        self.concat_layer = nn.Linear(2 * config.classifier_input_size, config.hidden_size)
+        self.attention = LocalAttention(config.classifier_input_size)
+        self.concat_layer = nn.Linear(config.classifier_input_size, config.hidden_size)
+        layers = [
+            nn.Sequential(activations[config.activation], nn.Linear(config.hidden_size, config.hidden_size))
```

Comment:
Create function `get_activation` which checks the property and returns correct function or raises an exception.

Reply:
Also, as far as I know, you don't need to `nn.Sequential` here. You can use it one time on list of modules.